package.json

```
@@ -27,7 +27,7 @@
   ],
   "author": "OpenFGA",
   "dependencies": {
-    "@openfga/sdk": "^0.1.0",
+    "@openfga/sdk": "0.1.1",
```

```suggestion
    "@openfga/sdk": "^0.1.1",
```

Do not pin dependencies on libraries unless you have to. This allows people to benefit from latest non-breaking changes even if you didn't update the dependencies.

---

updated